econml/dml.py

```diff
@@ -271,29 +276,91 @@ def predict(self, X):
                 return reshape_treatmentwise_effects(prediction,
                                                      self._d_t, self._d_y)
         self.bias_part_of_coef = fit_cate_intercept
+        self.fit_cate_intercept = fit_cate_intercept
         super().__init__(model_y=FirstStageWrapper(model_y, is_Y=True),
                          model_t=FirstStageWrapper(model_t, is_Y=False),
                          model_final=FinalWrapper(),
                          discrete_treatment=discrete_treatment,
                          n_splits=n_splits,
                          random_state=random_state)
 
+    @property
+    def original_featurizer(self):
+        return super().model_final._original_featurizer
+
     @property
     def featurizer(self):
+        # NOTE This is used by the inference methods and has to be the overall featurizer. intended
+        # for internal use by the library
         return super().model_final._featurizer
 
     @property
     def model_final(self):
+        # NOTE This is used by the inference methods and is more for internal use to the library
+        return super().model_final._model
+
+    def model_cate(self):
```

Comment:
1. Should this be decorated with `@property`?  
2. What's the distinction between `model_final` and `model_cate`?

Reply:
consistency with the DR Learner interface

Reply:
Done with adding property